lib/matplotlib/dviread.py

```diff
@@ -1017,16 +1017,16 @@ def find_tex_file(filename, format=None):
     if format is not None:
         cmd += ['--format=' + format]
     cmd += [filename]
-    try:  # Below: strip final newline.
-        result = cbook._check_and_log_subprocess(cmd, _log)[:-1]
+    try:
+        result = cbook._check_and_log_subprocess(cmd, _log)
     except RuntimeError:
         return ''
     if os.name == 'nt':
         # On Windows only, kpathsea appears to use utf-8 output(?); see
         # __win32_fputs in the kpathsea sources and mpl issue #11848.
-        return result.decode('utf-8')
+        return result.decode('utf-8')[:-2]  # Strip final '\r\n'.
```

Comment:
wouldn't it be more clear to just `.rstrip()` in both cases?

Reply:
Well, technically filenames can end with spaces... not in this case as we're always passing in the extension, but I think "strip the final newlines" (at least `rstrip("\r\n")`) is clearer in intent.

Reply:
I wouldn't trust any command line tool with trailing whitespace. 😄 

Anyway `.rstrip("\r\n")` would be more safe. Just in case the might be more or less newlines in future versions.

Reply:
done